conan/internal/api/uploader.py

```diff
@@ -271,16 +271,17 @@ def gzopen_without_timestamps(name, fileobj, compresslevel=None):
     return t
 
 
-def compress_files(files, name, dest_dir, compresslevel=None, ref=None):
+def compress_files(files, name, dest_dir, compresslevel=None, ref=None, recursive=False):
     t1 = time.time()
     # FIXME, better write to disk sequentially and not keep tgz contents in memory
     tgz_path = os.path.join(dest_dir, name)
-    ConanOutput(scope=str(ref)).info(f"Compressing {name}")
+    if ref:
+        ConanOutput(scope=str(ref)).info(f"Compressing {name}")
```

Comment:
```suggestion
    if ref:
        ConanOutput(scope=str(ref) if ref else None).info(f"Compressing {name}")
```
Lets fix the scope, but better not suppresss the message, as it is informative and for ``conan_cache_save.tgz`` it can be long time